tests/D2L.CodeStyle.Analyzers.Test/Async/Generator/AsyncToSyncMethodTransformerTests.cs

```diff
@@ -175,6 +175,15 @@ internal sealed class AsyncToSyncMethodTransformerTests {
 		Assert.AreEqual( "[Blocking] void Bar() { Func<int, Task> baz = quux => fred.Delay( 2*y ); }", actual.Value.ToFullString() );
 	}
 
+	[Test]
+	public void WrapInTaskRun() {
+		var actual = Transform( @"[GenerateSync] async Task BarAsync() { string baz = response.Content.ReadAsStringAsync(); }" );

```

Comment:
```suggestion
		var actual = Transform( @"[GenerateSync] async Task BarAsync() { string baz = await response.Content.ReadAsStringAsync(); }" );
```
right?
Maybe we should spend some time some day to check that the input things compile...

Reply:
Good idea